docs/source/release_notes.rst

```diff
@@ -53,6 +54,7 @@ Release Notes
 
     **Breaking Changes**
         * Removed ``has_searched`` property from ``AutoMLSearch`` :pr:`1647`
+        * Removed support for `np.random.RandomState` in EvalML :pr:`1727`
```

Comment:
I think it could be nice to provide a little more color here explaining where it was used before and what the standard is now (using random seed).